### ISSUE TITLE
Negative date value on recently added datarequests

### ISSUE THREAD

**Issue by andela-onnenanya** [CodeForAfricaLabs/ckanext-datarequests#4]
When a user creates a new data request, the created date displayed has a negative value. It takes about 24hrs to return to a positive value. I think this happens because minutes and seconds were not put into consideration in displaying the date.

![image](https://user-images.githubusercontent.com/26964897/30420646-271db7c2-9932-11e7-92a2-eda917db29c0.png)

                                  ||
                                  ||
                                  \/
![image](https://user-images.githubusercontent.com/26964897/30420656-2ce1372e-9932-11e7-97bc-60a5b68b0d62.png)

**Comment by andela-onnenanya** [comment]
I also observed the same behavior in comments on recently added data requests. 

![image](https://user-images.githubusercontent.com/26964897/30430773-348c0d2e-9954-11e7-9216-6f67baf5d5be.png)

**Comment by dmkitui** [comment]
Interested in looking to resolve this issue.

**Comment by shemogumbe** [comment]
This issue is likely to be in progress by @andela-onnenanya

**Comment by andela-onnenanya** [comment]
@shemogumbe Not really. @dmkitui can go ahead and work on the issue, I'm not currently working on it. I want to pick another task.

@dmkitui But you have to ensure the error exists in ckan-2.6.3 before you go forward.

**Comment by shemogumbe** [comment]
@dmkitui Try it out on 2.6.3 and if it exists go ahead and do it

**Comment by dmkitui** [comment]
@shemogumbe and @andela-onnenanya Will revert with a confirmation of state of the error on ckan-2.6.3 and proceed if it's replicable.

**Comment by andela-onnenanya** [comment]
@dmkitui I just checked on my own instance of CKAN-2.6.3, the error is replicable.

**Comment by dmkitui** [comment]
@andela-onnenanya Confirmed. Will be digging into this bug shortly.

<img width="1232" alt="screen shot 2017-09-19 at 11 24 31" src="https://user-images.githubusercontent.com/28535137/30582656-33259afe-9d2d-11e7-954c-701e64e5cffb.png">

**Comment by dmkitui** [comment]
I have interrogated this issue and it stems from a double application of timezone information on the current time and the time a datarequest or comment on a data request was created- during the calculation of the **time-since-created**. Its solution lies in editing _helpers.py_ and _formatters.py_ both located in the **ckan** repo. I have created a PR for the same on the andela/ckan repo, but for some reason it doesnt look right, because it ends up showing on the *ckan/ckan* repo. Kindly have a look and advise. 
https://github.com/ckan/ckan/pull/3819

**Comment by andela-onnenanya** [comment]
@dmkitui I believe you can still do same in the ckanext-openafrica helpers.py. I'll try to see if I can get any result in the next couple of minutes.
